### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ch-one**
Updated documentation to match version 0.5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,7 +62,8 @@ extension IPOrigin: JSONMappable {
     }
 }
 
-let request = NetworkRequest(path: "/ip", baseURLKey: baseURLKey)
+let url = URL(string: "https://httpbin.org/ip")!
```
Comment (lightsprint09) [inline README.md]: I would prefer the following 

```swift
let url = URL(string: "https://httpbin.org")!
let request = URLRequest(path: "/ip", baseURL: url)
```

Comment (ch-one) [inline README.md]: When looking into the documentation, I can't find a definition of an inizializer on URLRequest, that takes a baseURL and a path. Does it really exist?

Comment (lightsprint09) [inline README.md]: There is a custom initializer provided by us ;)

https://github.com/dbsystel/DBNetworkStack/blob/develop/Source/URLRequestConvertible.swift#L45

Comment (ch-one) [inline README.md]: 👍

**Pull request merged**